src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -3,12 +3,12 @@
         <!-- Refactoring needed to toogle view -->
         <div class="col-md-12 col-sm-12 col-xs-12" *ngIf="true">
             <div *ngIf="loggedIn" class="">
-                <textarea rows="2" type="text" id="wi-comment-add-comment" class="form-control width100" [(ngModel)]="comment.attributes.body"
-                    [style.height]="descHeight" [style.resize]="descResize" (keydown.enter)="createComment($event)" (keyup.enter)="preventDef($event)"
-                    placeholder="Add a comment here...">
-                </textarea>
+                <div almEditable id="wi-comment-add-comment" class="form-control width100"
+                    (focus)="resetCommentDraft($event)" (blur)="resetCommentDraft($event)"
+                    (keydown.enter)="createComment($event)" (keyup.enter)="preventDef($event)"
+                    [textContent]="'Add a new comment...'" #comment></div>
```

Comment:
contenteditable div doesn't have placeholders, but we can add styles etc afterwards to make it look like placeholder. Considering extending almEditable directive for that purpose.

For now the resetCommentDraft handles it such a way the placeholder can't be posted as comment (apart from the catch by @sanbornsen about when it's focused by default).